bears/python/YapfBear.py

```diff
@@ -0,0 +1,159 @@
+from coalib.bearlib.abstractions.Linter import linter
+from yapf.yapflib.yapf_api import FormatFile
+from coalib.results.Result import Result
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from coalib.results.Diff import Diff
+from coalib.misc.ContextManagers import prepare_file
+
+
+@linter(executable='yapf',
+        output_format='corrected',
+        result_message='Formatting can be improved.')
+class YapfBear:
+    """
+    Check and correct formatting of python Code using yapf utility.
+    """
+    LANGUAGES = {"Python"}
```

Comment:
could you add requirements too please? this bear has a PythonRequirement, piing me for help